src/Illuminate/Foundation/Console/stubs/convert-to-api/AlwaysAcceptJsonResponses.stub

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+
+class AlwaysAcceptJsonResponses
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure(\Illuminate\Http\Request): (\Illuminate\Http\Response|\Illuminate\Http\RedirectResponse)  $next
+     * @return \Illuminate\Http\Response|\Illuminate\Http\RedirectResponse
+     */
+    public function handle(Request $request, Closure $next)
+    {
+        return $next(tap($request, function ($request) {
+            $request->headers->set('Accept', 'application/json');
+        }));
+    }
+}
```

Comment:
This can be an actual class in the framework that can be copied over. No need to make it a stub.